e2e/tests/dropdownMoveFocusOnTab-test.ts

```diff
@@ -0,0 +1,33 @@
+import { Dropdown } from '@stardust-ui/react'
+
+describe('Dropdown', () => {
+  const triggerButton = `.${Dropdown.slotClassNames.triggerButton}`
+  const nextButton = '#next-button'
```

Comment:
lets declare these ids in the `example` file and consume them by `import` - this will ensure that this selector IDs are defined in the single place